modules/core/quick/tests/modules/farm_quick_test/src/Plugin/QuickForm/Test.php

```diff
@@ -65,7 +65,7 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
         [
           'measure' => 'count',
           'value' => $value,
-          'units' => $this->t('tests'),
+          'units' => 'tests',
```

Comment:
Interesting.. I was curious about sending the result of `t('string')` (translatable markup) to `QuickStringTrait::trimString()` or `QuickStringTrait::prioritizedString()` and if that would break things, but it seems like it doesn't.

But now seeing this, I'm curious why did *this* change, did it break something? Is this something that needs to be supported? Maybe if you have a quick form that has a `unit` string hard-coded, but you also want that unit to be translated? (but also... translating here may not be a good idea. in this case the `unit` term content should be translated I think..)

> \Drupal\Core\StringTranslation\TranslatableMarkup An object that, when cast to a string, returns the translated string.
https://api.drupal.org/api/drupal/core%21includes%21bootstrap.inc/function/t/10.0.x

Looking at the docs on this maybe it is because the `QuickStringTrait` functions are casting the translatable markup to a string? My hunch is that this would work: `'units' => (string) $this->t('tests')`. That seems like a fair workaround. Mostly thinking out loud here, I don't think there is any problem!

Reply:
Yes it did break the test because the `\Drupal\Core\StringTranslation\TranslatableMarkup` is not a string.

> My hunch is that this would work: 'units' => (string) $this->t('tests').

Yes that would work, as well as `$this->t('tests')->toString()` I believe.